app/components/wallet-overview/wallet-overview.tsx

```diff
@@ -179,48 +179,60 @@ const WalletOverview: React.FC<Props> = ({
 
 export default WalletOverview
 
-const useStyles = makeStyles(({ colors }) => ({
-  container: {
-    backgroundColor: colors.whiteOrDarkGrey,
-    display: "flex",
-    flexDirection: "column",
-    marginHorizontal: 30,
-    marginBottom: 20,
-    borderRadius: 12,
-    padding: 15,
-    borderWidth: 1,
-    borderColor: colors.grey9,
-  },
-  loaderBackground: {
-    color: colors.loaderBackground,
-  },
-  loaderForefound: {
-    color: colors.loaderForeground,
-  },
-  displayTextView: {
-    display: "flex",
-    flexDirection: "row",
-    justifyContent: "space-between",
-  },
-  separator: {
-    height: 1,
-    backgroundColor: colors.lighterGreyOrBlack,
-    marginTop: 10,
-  },
-  currency: {
-    display: "flex",
-    flexDirection: "row",
-    alignItems: "center",
-    columnGap: 10,
-  },
-  hideableArea: {
-    alignItems: "flex-end",
-    marginTop: 20,
-  },
-  loaderContainer: {
-    flex: 1,
-    justifyContent: "flex-end",
-    alignItems: "flex-end",
-    height: 43,
-  },
-}))
+const useStyles = makeStyles(
+  ({ colors }, { isContentVisible }: { isContentVisible: boolean }) => ({
```

Comment:
that sounds quite hacky. I think we should try to solve this with a pure CSS way, maybe with some min/max margin, or something along those lines?

Reply:
Yeah, using flexbox `alignItems: center` was sufficient. I have removed the unnecessary complexity.